assets/css/style.css

```diff
@@ -35,15 +31,43 @@
   align-items: center;
 }
 
+.menu {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .menu {
+    display: inherit;
+    gap: 1rem;
+    align-items: center;
+    list-style: none;
+  }
+
+  .top_header {
+    padding: 1.4rem 8rem;
+  }
+
+  .toggle {
+    display: none;
+  }
+}
+
 .headline {
   background: #1c1a19;
   padding: 2rem;
-  height: 80vh;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url("../images/Illustration.png");
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

There are multiple ways of achieving this, one of them is creating a class `"flex"`:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓